### PR TITLE
Adds 'c dc rebuild [service]'

### PR DESCRIPTION
**Pull request by smebberson**
This PR adds a new command: `c dc rebuild [service]`. It will rebuild and then restart a running service.

## Setup

- [x] Check out this branch.
- [x] Execute `npm link` to globally link this version of `c`.

## Testing

- [x] Get a new terminal window into the project you intended to test with (otherwise weird issues).
- [x] Start a project.
- [x] Execute `c dc rebuild` and you should get a nice error message.
- [x] Execute `c dc rebuild app` and make sure the appropriate service is rebuilt and restarted.

## Notes

- When finished, make sure you run `npm unlink`, and then replace the terminal windows which you used for testing this.

### CODE REVIEW

File: bin/lib/c.js [hunk 1]
```diff
@@ -64,9 +65,48 @@ const throwErr = (err) => {
...
+
+const composeUp = (service) => {
+
+    // We're bringing up just one container.
+    let upCommand = 'docker-compose up -d';
+
+    // We need to scale some containers.
+    /* eslint-disable padded-blocks */
+    if (['app', 'consul'].includes(service)) {
+        upCommand += ` --scale ${service}=2`;
```
Comment (allanchau) [inline bin/lib/c.js]: Should the number of services be configurable?

File: bin/lib/c.js [hunk 2]
```diff
@@ -64,9 +65,48 @@ const throwErr = (err) => {
...
+
+    /* eslint-disable padded-blocks */
+    if (program) {
+        program.help();
+    }
+    /* eslint-enable padded-blocks */
+
+};
+
+const composeUp = (service) => {
```
Comment (allanchau) [inline bin/lib/c.js]: Missing some jsdoc here.

**Comment by allanchau** [comment]
@smebberson This is working well, would be good to add `c dc logs [service]` too. Would that be hard to add to this PR?

File: bin/lib/c.js (hunk 1 above)
Comment (smebberson) [reply bin/lib/c.js]: @allanchau, yeah, I intend support this through some `json` in the `devops` folder. I'll implement this in another PR.

File: bin/lib/c.js (hunk 2 above)
Comment (smebberson) [reply bin/lib/c.js]: Good catch, fixed that :)

**Comment by smebberson** [comment]
@allanchau, updated `c dc logs` to `c dc logs <service>` :)

**Comment by smebberson** [comment]
@allanchau, can you please take another look at this one?

**Pull request merged**